include/exadg/utilities/parameter_handler.h

```diff
@@ -0,0 +1,29 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2022 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef INCLUDE_EXADG_UTILITIES_PARAMETER_HANDLER_H_
+#define INCLUDE_EXADG_UTILITIES_PARAMETER_HANDLER_H_
+
+#include <deal.II/base/parameter_handler.h>
+
+#include <exadg/utilities/enum_patterns.h>
+
+#endif /*INCLUDE_EXADG_UTILITIES_PARAMETER_HANDLER_H_*/
```

Comment:
I do not get the purpose of this file.

Reply:
It is important to include the two headers in this exact order. To avoid mixing this up and also to be able to have a clean interface for the user, I added the file.

Reply:
It would make more sense in my opinion to include `dealii/base/parameter_handler.h` (with a suitable comment) in `<exadg/utilities/enum_patterns.h>`. The current choice with a new (empty) header file looks like needless complexity to me and is very hard to understand in my opinion.

Reply:
If it is OK for you to include `#include <exadg/utilities/enum_patterns.h>` instead of a `parameter_handler.h` its OK for me.

Reply:
You mean including dealii/parameter_handler additionally in other files in ExaDG will give an error?

Reply:
see comment https://github.com/exadg/exadg/pull/382/files#r1214137407

Reply:
No, it is still possible to include parameter handler. But if one wants to treat `enums` one has to include `enum_patterns.h` in addition. I think I will adjust it this way.

Reply:
I changed it in [c3f0a7d](https://github.com/exadg/exadg/pull/382/commits/c3f0a7daf7857d39dcd9474eb86602d5bde96c1f)